README.rst

```diff
@@ -429,6 +429,13 @@ Choose from the list of available fixers:
                          or function name and the
                          opening parenthesis.
 
+* **no_spaces_inside_ofsset**
+                         There MUST NOT be a space
```

Comment:
What about "There MUST NOT be spaces between offset square braces and its contained values."?